templates/wiki/page_in_use.html

```diff
@@ -0,0 +1,10 @@
+{% extends "wiki/base.html" %}
+{% block title %}Wiki Error{% endblock %}
+{% block og_title %}Wiki Error{% endblock %}
+{% block og_description %}{% endblock %}
+{% block content %}
+    <div class="uk-container uk-container-small">
+        <h1>The page you requested is currently being edited</h1>
+        <p>Please try again in a little bit when the lock has expired.</p>
+    </div>
+{% endblock %}
```

Comment:
Missing newline at end of file

Reply:
Fixed in latest commit (same with all other newline reviews).